src/Compilers/VisualBasic/Test/Syntax/Parser/VisualBasicParseOptionsTests.vb

```diff
@@ -335,7 +335,7 @@ Public Class VisualBasicParseOptionsTests
         symbols.Add("1", Nothing)
         Dim options = New VisualBasicParseOptions(preprocessorSymbols:=symbols)
 
-        options.Errors.Verify(Diagnostic(ERRID.ERR_ConditionalCompilationConstantNotValid).WithArguments("Identifier expected.", "1").WithLocation(1, 1))
+        options.Errors.Verify(Diagnostic(ERRID.ERR_ConditionalCompilationConstantNotValid).WithArguments(VBResources.ERR_ExpectedIdentifier, "1").WithLocation(1, 1))
```

Comment:
>VBResources.ERR_ExpectedIdentifier [](start = 105, length = 34)

Looks like the same bug in VisualBasicParseOptions.ValidateOptions. Applies to all changes in this file.